html/elements/menulist.json

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
This whole file needs to be renamed to menuitem.json and the object should be html.elements.menuitem. The MDN URL is wrong, too.

Reply:
Fixed.